dev-util/kdstatemachineeditor/kdstatemachineeditor-1.2.4.ebuild

```diff
@@ -0,0 +1,50 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-utils kde5-functions
+
+DESCRIPTION="Framework for creating Qt State Machine metacode using graphical user interfaces"
+HOMEPAGE="https://github.com/KDAB/KDStateMachineEditor"
+
+if [[ ${PV} == 9999 ]] ; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/KDAB/KDStateMachineEditor.git"
+else
+	SRC_URI="https://github.com/KDAB/KDStateMachineEditor/releases/download/v${PV}/${P}.tar.gz"
+	KEYWORDS="~amd64"
+fi
+
+LICENSE="GPL-2+"
+IUSE="doc test"
+SLOT="0"
+
+RDEPEND="
+	$(add_qt_dep qtcore)
+	$(add_qt_dep qtdeclarative 'widgets')
+	$(add_qt_dep qtgui)
+	$(add_qt_dep qtnetwork)
+	$(add_qt_dep qtwidgets)
+"
+
+DEPEND="${RDEPEND}
+	doc? (
```

Comment:
this line and below: all of it is in `BDEPEND` instead of `DEPEND` in EAPI 7. https://blogs.gentoo.org/mgorny/2018/05/03/the-ultimate-guide-to-eapi-7/